src/ctap/storage.rs

```diff
@@ -577,6 +577,11 @@ pub fn enterprise_attestation(env: &mut impl Env) -> Result<bool, Ctap2StatusCod
 
 /// Marks enterprise attestation as enabled.
 pub fn enable_enterprise_attestation(env: &mut impl Env) -> Result<(), Ctap2StatusCode> {
+    if attestation_private_key(env).unwrap_or(None).is_none()
+        || attestation_certificate(env).unwrap_or(None).is_none()
```

Comment:
Why unwrap? I don't think we want to hide storage errors.

```rust
if attestation_private_key(env)?.is_none() || attestation_certificate(env)?.is_none() {
```

Reply:
Ah yes, of course.